### ISSUE TITLE
LibWeb/HTML: incorrect behavior on select()

### ISSUE THREAD

**Issue by mobounya** [LadybirdBrowser/ladybird#1446]
when calling select() on an input element it only works for the first time, clicking somewhere else and calling select() again does not work unless you change the text on the element.

incorrect behavior on Ladybird:
![ladybird](https://github.com/user-attachments/assets/4f99504d-1d92-4fc0-b887-b70c7adeefd7)

correct behavior on google chrome:
![chrome](https://github.com/user-attachments/assets/a224b3a2-46d9-4edc-b18f-7561d9ae2e21)

HTML code:
```
<script>
    let mySelect = () => {
        let foo = document.getElementById("foo");
        foo.select();
    }
</script>

<input type="text" id="foo"/>
<button style="padding: 15px" id="button" onclick="mySelect()">Select</button>
```

**Comment by ronak69** [comment]
Notice how prior to typing, the box is not vertically centerd with respect to the button box.

Also, tabbing more than two times (focus going past the button) freezes the app?

**Comment by mobounya** [comment]
yes both points are true, I guess they should be two more separate issues ?
